index.js

```diff
@@ -1,18 +1,60 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector('#header')
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "purple"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const travelerInfo = document.querySelector('div#profile h2')
+travelerInfo.innerText = traveler.name
+
+const travelerNickname = document.querySelector('div#profile em')
+travelerNickname.innerText = traveler.nickname
```

Comment:
👍  Good! You can also use `textContent` instead of `innerText`. The main difference is that `textContent` doesn't consider the CSS whereas `innerText` does. [Take a look at this stackoverflow post about it](https://stackoverflow.com/questions/35213147/difference-between-textcontent-vs-innertext)